.gitignore

```diff
@@ -1,17 +1,12 @@
 # Backend
 */db.sqlite3
-engine/oncall_dev.db
+engine/*.db
 *.pyc
 venv
 .python-version
-.env
-.env_hobby
-.env.dev
```

Comment:
I think `.env.dev` should be ignored too, so `.env` is for running OSS hobby, `.env.dev` for running local dev environment